sql/lex.h

```diff
@@ -317,8 +317,9 @@ SYMBOL symbols[] = {
   { "INT8",		SYM(BIGINT)},
   { "INTEGER",		SYM(INT_SYM)},
   { "INTERVAL",		SYM(INTERVAL_SYM)},
-  { "INVISIBLE",        SYM(INVISIBLE_SYM)},
   { "INTO",		SYM(INTO)},
+  { "INVISIBLE",        SYM(INVISIBLE_SYM)},
```

Comment:
lets leave the misording of INVISIBLE to a different problem. Someone fixing earlier and merging up may end up with a double-invisible.